### DISCUSSION TITLE
Tell us more

### DISCUSSION THREAD

**Discussion by simonfossom** [rajatkulkarni95/octarine-feedback#3] [General]
Hi there 👋

Looks very nice.

What stack do you use?
And do you plan to open-source?
Why are you building it with all other products in this space?

Cheers!

**Comment by rajatkulkarni95** [comment]
Heyo!

Thanks for trying the app out!

> What stack do you use?

React with Typescript/Tailwind/Vite, Rust for the file system handling, Tauri for the bundling

> And do you plan to open-source?

Not at the moment. There's plans to monetise it (additional features for a one-time payment), and I don't really have a way forward with open source for that model.

> Why are you building it with all other products in this space?

Used to use Obsidian and Notepad++. The former was clunky (this was before their UI refresh, which I think is neat), too much reliance on 3rd party plugins to get shit done, and generally not my vibe.

Notion was a neat alternative, but it started becoming slow and web/databases was weird. Still a fan of `Bear Notes` and their app is neat, but wanted to scratch an itch and had a few things in mind.

---

How are you liking the app so far? How did you learn about it?

**Reply by StefaniaLuisa** [reply]
Found the app recently through a reddit chat, looks really promising and excited to check it out, like the previous comment, the pro price is a bit steep, would have immediately purchased for $20 and have seen it as an investiment to a promising app, but hard to shell $50 when still in beta - totally get that you are doing this all yourself and you deserve to be paid for your labor, just my initial feedback upon seeing the price, maybe worth thinking about a monthly subscription rate as well as this "lifetime" version.   I will continue to use and evalute - I am hopeful to eventually purchase the Pro! Thank you!

**Reply by r0bbie** [reply]
> > And do you plan to open-source?

> 
> Not at the moment. There's plans to monetise it (additional features for a one-time payment), and I don't really have a way forward with open source for that model.

Just following up on the open source conversation, as was looking if there'd been any questions on that :)

Firstly, loving Octarine, amazing job on it! Just discovered it yesterday and purchased a lifetime license already. I know there's a thousand notes apps out there, but this one really stands out UX-wise - seriously impressive attention to detail, taste, and sense of "craft". Excited to use it more.

On open source - totally understand there can be concerns around monetisation and certain open source licenses! Expectation for folks to pay to support development is of course reasonable, I guess the main concern I tend to have is I've been burnt before with note-taking apps that seem super promising and end up becoming abandonware..

Do you think something like the [Business Source License](https://en.wikipedia.org/wiki/Business_Source_License) or [Functional Source License](https://fsl.software) might be something you'd consider? (i.e. not open source, but source available - so potentially folks can still contribute, and it eventually "converts" to open source after a few years, offering some protection against lock-in and abandonware..)

Or even just some commitment that if you ever do decide to cease development you'd open source it then / at least disable the licensing server so people can use it indefinitely without getting locked out? (of the app! understand the files are just markdown 🙌)

Just some thoughts anyway :) appreciate this is always a tricky discussion when weighing up business realities etc too..

**Comment by vdbvr-wouter** [comment]
I've been using Octarine for about a month now, and I'm really liking it.
The Daily Desk feature is the thing I mostly use as a work diary and ToDo list.

Pricing is a bit steep, but I'm certainly evaluating it.
